bundles/org.pitest.pitclipse.ui/src/org/pitest/pitclipse/ui/view/PitView.java

```diff
@@ -33,20 +32,18 @@
 
     @Override
     public synchronized void createPartControl(Composite parent) {
-        try {
-            browser = new Browser(parent, SWT.NONE);
-            browser.addProgressListener(new PitUiUpdatePublisher(browser));
-        } catch (SWTError e) {
-            MessageBox messageBox = new MessageBox(parent.getShell(), SWT.ICON_ERROR | SWT.OK);
-            messageBox.setMessage("Browser cannot be initialized.");
-            messageBox.setText("Exit");
-            messageBox.open();
-        }
+        PitclipseUiUtils.safeExecute
+            (() -> {
+                browser = new Browser(parent, SWT.NONE);
+                browser.addProgressListener(new PitUiUpdatePublisher(browser));
+            },
+            parent,
+            "Browser cannot be initialized.");
```

Comment:
I'd like the popup to explicitely tell that the error is related to the summary view, could we say something like "Failed to create the Pit Summary view: browser cannot be initialized" or would it be too long?

Reply:
No, I think it would be fine!